test/specs/products.js

```diff
@@ -22,22 +23,32 @@ describe('Products Page Tests', () => {
             expect(await browser.getUrl()).toContain(ProductsPage.url);
         });
 
-        it('should have the "SWAGLABS" logo in header', async () => {
+        it('should have the "SWAGLABS" logo in primary header', async () => {
 
             expect(await ProductsPage.appLogo.isDisplayed()).toBe(true);
         });
 
-        it('should have the Burger Menu button in header', async () => {
+        it('should have the Burger Menu button in primary header', async () => {
 
             expect(await ProductsPage.btnBurgerMenu.isDisplayed()).toBe(true);
         });
 
-        it('should have the Cart Icon in header', async () => {
+        it('should have the Cart Icon in primary header', async () => {
 
             expect(await ProductsPage.ShoppingCartIcon.isDisplayed()).toBe(true);
         });
 
-        it('should have the filter dropdown menu in header', async () => {
+        it("should have the 'PRODUCTS' text in secondary header", async () => {
+
+            expect(await InventoryPage.productsHeader.getText()).toEqual("PRODUCTS");
+        });
+
+        it('should have the Robot Peek in secondary header', async () => {
+
+            expect(await InventoryPage.robotPeek.isDisplayed()).toBe(true);
+        });
```

Comment:
You should combine these tests into one test since they are only expects.

Reply:
the header and the footer tests have been grouped under the "check Header UI elements", respectively ""check Footer UI elements" test cases